### ISSUE TITLE
please contact

### ISSUE THREAD

**Issue by krizka** [javascriptpoet/redux-tcomb-forms#1]
Sorry for the issue, just want to reach you. Please answer or skype kostya.kurilov )

**Comment by javascriptpoet** [comment]
sorry for delay. just saw raised issue. how can i help you.

**Comment by krizka** [comment]
Mm. I think somewhere I saw your post or message about you want to make some CRM on abstract objects with visual editor on it. If it is still actual - to collaborate on it.

**Comment by javascriptpoet** [comment]
yes. i'm slowly getting close to it. I love collaborating. I'm about to release a js framework that will allow composing whole apps. I have a very clear vision how my framework can be coupled with a gui interface to crate app boilerplates - types similar to react components. An app component typed/described in one js file and created thru visualization tools like connected graphs of visjs. The framework is based on reality abstraction - space, time, information flow. It is working. It cleared a lot of my confusion and fixed a lot of syntactic flows of js, at least for me. I think, the best is to wait a few weeks for me to debug it and put together some docs. Then, i would love your input. My goal is to create tools to allow close and efficient collaboration of all types of creative people with the goal of making increadable user interfaces - for humans by humans, away from stifling corps, to promote freedom of expression, to bring us all closer, to unite our minds.

**Comment by javascriptpoet** [comment]
i'm sorry for such a huge delay in my response. It is not due to a lack of interest. I simply dont use this as a comm channel. It will be changing soon. In meantime, heres my fb link. Please, connect with me and we can establish a channel over slack. Thats my prefered method. I'm very excited to have someone collaborating on my dreams. https://www.facebook.com/dmitry.shusterman
